### DISCUSSION TITLE
Create overview table with links for miller solutions listed in #212

### DISCUSSION THREAD

**Discussion by NikosAlexandris** [johnkerl/miller#609] [Ideas]
The large list of solutions in https://github.com/johnkerl/miller/issues/212#issuecomment-451508839 can be sorted and categorized, each solution tagged in a meaningful way. This could either go in the main ReadMe page of the repository, or else in a dedicated Wiki page. Or so. Thank you @aborruso.

**Comment by aborruso** [comment]
Hi @NikosAlexandris I have created this TSV https://gist.github.com/aborruso/009acf961fecf9cf22b693f90fdf0603

@johnkerl what do you think about Nikos proposal?

| title | URL |
| --- | --- |
| How to add new columns to CSVs with the same text in all lines | https://askubuntu.com/questions/1193534/how-to-add-new-columns-to-csvs-with-the-same-text-in-all-lines/1193544#1193544 |
| csv text-processing question | https://unix.stackexchange.com/questions/555643/csv-text-processing-question/555667#555667 |
| how to keep rows if there are a certain number of observations based on the first column? | https://unix.stackexchange.com/questions/558147/how-to-keep-rows-if-there-are-a-certain-number-of-observations-based-on-the-firs/558258#558258 |
| Add Column to CSV based on Filename | https://stackoverflow.com/questions/59422250/batch-script-add-column-to-csv-based-on-filename/59426724#59426724 |
| Concatenating columns of the same csv file to create a new column with a new heading | https://unix.stackexchange.com/a/558359/195582 |
| How can I replace strings in a given column of a table using awk and regex? | https://unix.stackexchange.com/a/558362/195582 |
| JSON -> csv creating header line and padding header if found empty field | https://unix.stackexchange.com/questions/507005/json-csv-creating-header-line-and-padding-header-if-found-empty-field/560690#560690 |
| How to convert yaml to csv file format? | https://askubuntu.com/questions/941477/how-to-convert-yaml-to-csv-file-format/1201270#1201270 |
| Shard CSV based on time values | https://stackoverflow.com/questions/59673777/shard-csv-based-on-time-values/59682909#59682909 |
| How to parse Stdout as a mix of CSV and JSON? | https://unix.stackexchange.com/questions/561467/how-to-parse-stdout-as-a-mix-of-csv-and-json/561576#561576 |
| How to use awk for joining two csv using a key column and write a another csv | https://unix.stackexchange.com/questions/560266/how-to-use-awk-for-joining-two-csv-using-a-key-column-and-write-a-another-csv/561580#561580 |
| How to extract column name (header) from a CSV file which contains the max value in a row? | https://unix.stackexchange.com/questions/217817/how-to-extract-column-name-header-from-a-csv-file-which-contains-the-max-value/562350#562350 |
| Need help to JOIN two CSV files | https://stackoverflow.com/questions/59913035/need-help-grepping-from-csv/59916638#59916638 |
| Applying sort -u to one specific column in a CSV file? | https://stackoverflow.com/questions/59924061/applying-sort-u-to-one-specific-column-in-a-csv-file/59946495#59946495 |
| compare the values of 2 files and merge the similar entries | https://stackoverflow.com/questions/60024879/compare-the-values-of-2-files-and-merge-the-similar-enteries/60025241#60025241 |
| How to split one column into two based on a certain character? | https://unix.stackexchange.com/questions/565537/how-to-split-one-column-into-two-based-on-a-certain-character/565541#565541 |
| SSV/CSV Manipulation: Calculate Ratios | https://unix.stackexchange.com/a/566055/195582 |
| Column transposition according to two values | https://unix.stackexchange.com/questions/566096/column-transposition-according-to-two-values/566098#566098 |
| Bash - Removing empty columns from .csv file | https://stackoverflow.com/questions/60098400/bash-removing-empty-columns-from-csv-file/60134937#60134937 |
| Merge 3 .csv into one file with different number of columns | https://superuser.com/questions/1523440/merge-3-csv-into-one-file-with-different-number-of-columns/1524270#1524270 |
| csv file - remove line feed in the middle of string | https://stackoverflow.com/questions/60177576/csv-file-remove-line-feed-in-the-middle-of-string/60178030#60178030 |
| How to get data for past hour, 6h, 24h from CSV log file with awk, sed or other? | https://stackoverflow.com/questions/60365724/how-to-get-data-for-past-hour-6h-24h-from-csv-log-file-with-awk-sed-or-other/60371907#60371907 |
| How to add a value of the variable at the beginning of a csv file | https://stackoverflow.com/questions/60379142/how-to-add-a-value-of-the-variable-at-the-beginning-of-a-csv-file/60390718#60390718 |
| Convert JSON array into CSV | https://unix.stackexchange.com/questions/570548/convert-json-array-into-csv/570571#570571 |
| How to combine TSV file | https://askubuntu.com/a/1217868/796708 |
| How to put pivot table using Shell script | https://stackoverflow.com/questions/61186811/how-to-put-pivot-table-using-shell-script/61204513#61204513 |
| Bash script to add double quotes in .CSV comma delimited file | https://stackoverflow.com/questions/61220101/bash-script-to-add-double-quotes-in-csv-comma-delimited-file/61244777#61244777 |
| How can I delete from a file every second and fourth comma separated word using sed? | https://unix.stackexchange.com/questions/581068/how-can-i-delete-from-a-file-every-second-and-fourth-comma-separated-word-using/581072#581072 |
| How to append additional text found in a duplicate entry to the original text | https://superuser.com/questions/1540589/how-to-append-additional-text-found-in-a-duplicate-entry-to-the-original-text/1543477#1543477 |
| Concatenate matching ids from two files, with other columns from both files | https://unix.stackexchange.com/questions/583795/concatenate-matching-ids-from-two-files-with-other-columns-from-both-files/583821#583821 |
| I want to convert list of json files to csv with only specific attributes | https://stackoverflow.com/questions/61819131/i-want-to-convert-list-of-json-files-to-csv-with-only-specific-attributes/61835027#61835027 |
| CSV Filtering a column with mixed data types | https://stackoverflow.com/a/62099705/757714 |
| (AWK) Csv separating same variables in columns into different files | https://unix.stackexchange.com/questions/590911/awk-csv-separating-same-variables-in-columns-into-different-files/591041#591041 |
| GROUP BY CSV columns in bash | https://stackoverflow.com/questions/62252413/group-by-csv-columns-in-bash/62258153#62258153 |
| vlookup using UNIX shell | https://unix.stackexchange.com/questions/592517/vlookup-using-unix-shell/592980#592980 |
| Querying csv files like sql | https://unix.stackexchange.com/questions/593939/querying-csv-files-like-sql/594193#594193 |
| Filling in blank spaces in the first column of a csv file using Bash / awk | https://unix.stackexchange.com/questions/601138/filling-in-blank-spaces-in-the-first-column-of-a-csv-file-using-bash-awk/602193#602193 |
| Log search within date range | https://unix.stackexchange.com/questions/604588/log-search-within-date-range/604736#604736 |
| Find value from one csv in another one (like vlookup) in bash (Linux) | https://stackoverflow.com/questions/10697724/find-value-from-one-csv-in-another-one-like-vlookup-in-bash-linux/63666127#63666127 |
| Manipulate csv file using bash script or python, converting certain columns into rows | https://askubuntu.com/a/1273008/796708 |
| Handle CSV data: Separate data with a comma | https://stackoverflow.com/questions/63999865/handle-csv-data-separate-data-with-a-comma/64012427#64012427 |
| How to “Partial-Transpose-and-Duplicate” rows in a CSV with a CSV command line tool | https://stackoverflow.com/questions/64220427/how-to-partial-transpose-and-duplicate-rows-in-a-csv-with-a-csv-command-line-t/64240640#64240640 |
| Csv trasform one row in 4 row | https://stackoverflow.com/questions/64322327/csv-trasform-one-row-in-4-row/64332420#64332420 |
| Grep the data only (-o) and the first column | https://unix.stackexchange.com/questions/615597/grep-the-data-only-o-and-the-first-column/617644#617644 |
| sorting a csv file with 3 columns | https://askubuntu.com/questions/1277136/sorting-a-csv-file-with-3-columns/1295780#1295780 |
| How do I print all lines of a file with duplicate values in a certain column | https://unix.stackexchange.com/questions/255563/how-do-i-print-all-lines-of-a-file-with-duplicate-values-in-a-certain-column/621876#621876 |
| Parsing through Bash | https://stackoverflow.com/questions/65208302/parsing-through-bash/65213118#65213118 |
| I need an efficient method to combine thousands of csv files | https://unix.stackexchange.com/questions/625168/i-need-an-efficient-method-to-combine-thousands-of-csv-files/625194#625194 |
| Compare csv files based on column value | https://stackoverflow.com/questions/65592814/compare-csv-files-based-on-column-value/65608417#65608417 |
| How to rearrange columns when you don't know their initial positions (UNIX, BASH) | https://superuser.com/questions/1618362/how-to-rearrange-columns-when-you-dont-know-their-initial-positions-unix-bash/1618583#1618583 |
| Sort multiple columns in csv including date stored as dd/mm/yyyy | https://unix.stackexchange.com/a/632204/195582 |
| How to get the pivot lines from two tab-separated files? | https://stackoverflow.com/questions/66121376/how-to-get-the-pivot-lines-from-two-tab-separated-files/66167539#66167539 |
| grep output from long to wide | https://unix.stackexchange.com/questions/634364/grep-output-from-long-to-wide/634376#634376 |
| Adding CSV layer to QGIS when X and Y fields are the same | https://gis.stackexchange.com/questions/388039/adding-csv-layer-to-qgis-when-x-and-y-fields-are-the-same/388042#388042 |
| How to split the data from two columns based on two different delimiters in a Redshift database? | https://stackoverflow.com/questions/66359957/how-to-split-the-data-from-two-columns-based-on-two-different-delimiters-in-a-re/66412025#66412025 |
| How to convert a CSV (with quote) to TSV? | https://stackoverflow.com/questions/66422944/how-to-convert-a-csv-with-quote-to-tsv/66435748#66435748 |
| converting space separated file into csv format in linux | https://stackoverflow.com/questions/66456419/converting-space-separated-file-into-csv-format-in-linux/66477278#66477278 |
| Bash to send variable value to csv? | https://unix.stackexchange.com/questions/650580/bash-to-send-variable-value-to-csv/650724#650724 |
| awk extract columns that meet condition | https://unix.stackexchange.com/a/651415/195582 |
| Is it possible to rename multiple columns of a CSV to empty columns when using miller? | https://stackoverflow.com/a/68452078/757714 |
| How to merge multiple Mac excel files all with headers that are the same but out of order | https://superuser.com/questions/1666750/how-to-merge-multiple-mac-excel-files-all-with-headers-that-are-the-same-but-out/1666899#1666899 |
| SED delete csv column if exists - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/492434/sed-delete-csv-column-if-exists/492534#492534 |
| sed - Parsing CSV that has multiple line values with carriage returns - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/486655/parsing-csv-that-has-multiple-line-values-with-carriage-returns/491748#491748 |
| bash - CSV file: Group and sum values based on first pattern of each value - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/472858/csv-file-group-and-sum-values-based-on-first-pattern-of-each-value/491732#491732 |
| awk - Organizing data by a header - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/476023/organizing-data-by-a-header/491731#491731 |
| linux - Add a pattern before the delimiter - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/484274/add-a-pattern-before-the-delimiter/491730#491730 |
| text processing - Output a header label in data field in miller - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/430199/output-a-header-label-in-data-field-in-miller/492541#492541 |
| text processing - Stack data from multiple delimited files into one, with variable columns - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/238046/stack-data-from-multiple-delimited-files-into-one-with-variable-columns/492859#492859 |
| awk - Convert data from LDIF file to CSV - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/457091/convert-data-from-ldif-file-to-csv/492863#492863 |
| text processing - cut an array column and join an another column to a csv file - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/318446/cut-an-array-column-and-join-an-another-column-to-a-csv-file/492865#492865 |
| text processing - Join two files based on a common field - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/493551/join-two-files-based-on-a-common-field/493629#493629 |
| text processing - Compare columns in 2 files and put information according to file 1 - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/493552/compare-columns-in-2-files-and-put-information-according-to-file-1/493627#493627 |
| csv - Getting file as rows and columns data matrix - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/494259/getting-file-as-rows-and-columns-data-matrix/494273#494273 |
| extracting numbers whose lengths are fixed from text file - Ask Ubuntu | https://askubuntu.com/questions/1109590/extracting-numbers-whose-lengths-are-fixed-from-text-file/1109598#1109598 |
| shell script - How remove duplicate names and print array after unique names - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/494778/how-remove-duplicate-names-and-print-array-after-unique-names/494801#494801 |
| awk - How can I remove repeated lines taking into consideration the text from just a column? - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/494803/how-can-i-remove-repeated-lines-taking-into-consideration-the-text-from-just-a-c/494874#494874 |
| SQL operation on csv file using bash or shell - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/495005/sql-operation-on-csv-file-using-bash-or-shell/495026#495026 |
| text processing - command to layout tab separated list nicely - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/7698/command-to-layout-tab-separated-list-nicely/495416#495416 |
| software rec - Is there a robust command line tool for processing csv files? - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/7425/is-there-a-robust-command-line-tool-for-processing-csv-files/495417#495417 |
| awk - Merging two csv files, can't get rid of newline - Stack Overflow | https://stackoverflow.com/questions/54273438/merging-two-csv-files-cant-get-rid-of-newline/54274740#54274740 |
| python - Convert entry log (csv) to list of names based on whether they entered on each day - Stack Overflow | https://stackoverflow.com/questions/54273622/convert-entry-log-csv-to-list-of-names-based-on-whether-they-entered-on-each-d/54275979#54275979 |
| python - Split large csv file into several based on condition - Stack Overflow | https://stackoverflow.com/questions/54277980/split-large-csv-file-into-several-based-on-condition/54280028#54280028 |
| command line - Move specific column from csv file in front (select column by name) - Ask Ubuntu | https://askubuntu.com/questions/1109777/move-specific-column-from-csv-file-in-front-select-column-by-name/1111476#1111476 |
| Need to replace specific text in CSV file using batch script - Stack Overflow | https://stackoverflow.com/questions/54312302/need-to-replace-specific-text-in-csv-file-using-batch-script/54314203#54314203 |
| text processing - Converting from field: value format to CSV - Stack Overflow | https://stackoverflow.com/questions/54317248/converting-from-field-value-format-to-csv/54317558#54317558 |
| csv - How to re-pivot data in bash? - Stack Overflow | https://stackoverflow.com/questions/54353222/how-to-re-pivot-data-in-bash/54356470#54356470 |
| bash - Add trailing commas to ensure .csv rows have same number of columns - Stack Overflow | https://stackoverflow.com/questions/54363812/add-trailing-commas-to-ensure-csv-rows-have-same-number-of-columns/54368097#54368097 |
| text processing - To get average of columns for all the entries whose first column entries are duplicate - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/497716/to-get-average-of-columns-for-all-the-entries-whose-first-column-entries-are-dup/497884#497884 |
| text processing - Removing duplicate entries and replacing it with comma \| Bash - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/498132/removing-duplicate-entries-and-replacing-it-with-comma-bash/498150#498150 |
| text processing - How to generate a pivotal table? - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/499352/how-to-generate-a-pivotal-table/499444#499444 |
| bash - delete all line after a specific date - Stack Overflow | https://stackoverflow.com/questions/54594951/delete-all-line-after-a-specific-date/54596152#54596152 |
| csv - Remove rows based on values in a column - Stack Overflow | https://stackoverflow.com/questions/54661258/remove-rows-based-on-values-in-a-column/54664933#54664933 |
| text processing csv file, truncate digits without rounding - Stack Overflow | https://stackoverflow.com/questions/54679856/text-processing-csv-file-truncate-digits-without-rounding/54680559#54680559 |
| csv - What's the best practice to scan a text file on linux for quality checking (pattern matching)? - Stack Overflow | https://stackoverflow.com/questions/54678054/whats-the-best-practice-to-scan-a-text-file-on-linux-for-quality-checking-patt/54680780#54680780 |
| shell script - Extract data from CSV file, where some fields have comma in field value - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/501550/how-to-extract-fields-from-a-csv-i-e-comma-separated-where-some-of-the-fields-ha/501635#501635 |
| text processing - Output to last column depending on first column if is greater, less than or equal to 0 - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/502368/output-to-last-column-depending-on-first-column-if-is-greater-less-than-or-equa/502377#502377 |
| powershell - Combining Multiple CSV Files - Stack Overflow | https://stackoverflow.com/questions/17752072/combining-multiple-csv-files/54855980#54855980 |
| text processing - How could I (painlessly) split or reverse "Last, First" within a record in Miller? - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/504891/how-could-i-painlessly-split-or-reverse-last-first-within-a-record-in-mille/504978#504978 |
| command line - comparing the content and produce new file - Ask Ubuntu | https://askubuntu.com/questions/1122680/comparing-the-content-and-produce-new-file/1124822#1124822 |
| merge 2 csv files via batch - Stack Overflow | https://stackoverflow.com/questions/55124218/merge-2-csv-files-via-batch/55125480#55125480 |
| How can I count the rows of a CSV with only the linux terminal? - Stack Overflow | https://stackoverflow.com/questions/55149498/how-can-i-count-the-rows-of-a-csv-with-only-the-linux-terminal/55151056#55151056 |
| shell script - creating new columns - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/507719/creating-new-columns/507778#507778 |
| text processing - Delete multiple columns using awk or sed - Ask Ubuntu | https://askubuntu.com/questions/1127670/delete-multiple-columns-using-awk-or-sed/1127741#1127741 |
| macos - Copy First Column to New Column (Renamed Header) at End in CSV with Mac Terminal - Stack Overflow | https://stackoverflow.com/questions/55368375/copy-first-column-to-new-column-renamed-header-at-end-in-csv-with-mac-terminal/55371953#55371953 |
| How to format csv into multiline newline delimited plain text output using csvkit - Stack Overflow | https://stackoverflow.com/questions/51233041/how-to-format-csv-into-multiline-newline-delimited-plain-text-output-using-csvki/55529867#55529867 |
| linux - Getting values from comma seperated file - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/511658/getting-values-from-comma-seperated-file/511706#511706 |
| text processing - How to compare two CSV files and display unique records? - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/511773/how-to-compare-two-csv-files-and-display-unique-records/511831#511831 |
| text processing - Is there a way to parse multiple lines to a single line by merging duplicates and tabbing non-duplicates? - Stack Overflow | https://stackoverflow.com/questions/55602163/is-there-a-way-to-parse-multiple-lines-to-a-single-line-by-merging-duplicates-an/55627219#55627219 |
| text processing - how to rename output of split command to match the first word in each line? - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/512169/how-to-rename-output-of-split-command-to-match-the-first-word-in-each-line/512178#512178 |
| Split huge CSV by columns with Miller - Stack Overflow | https://stackoverflow.com/questions/55685497/split-huge-csv-by-columns-with-miller/55687128#55687128 |
| regex - Check if value of column 1 exists anywhere in column 2, write result to column 3 (CSV document) - Stack Overflow | https://stackoverflow.com/questions/55743810/check-if-value-of-column-1-exists-anywhere-in-column-2-write-result-to-column-3/55750730#55750730 |
| Page not found - Super User | https://superuser.com/questions/1432161/how-to-filter-value-in-csv-with-batch-and-show-the-output-to-txt/1432211#1432211 |
| bash - Reshaping from wide to long format - Stack Overflow | https://stackoverflow.com/questions/41842304/reshaping-from-wide-to-long-format/56073938#56073938 |
| linux - Pivot file values - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/520031/pivot-file-values/520468#520468 |
| Parsing csv which contains json - Stack Overflow | https://stackoverflow.com/questions/56257428/parsing-csv-which-contains-json/56261697#56261697 |
| text processing - Merging csv files with Key columns - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/520492/merging-csv-files-with-key-columns/520513#520513 |
| linux - How to convert a 3 column csv file into a table (or matrix) - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/522046/how-to-convert-a-3-column-csv-file-into-a-table-or-matrix/522550#522550 |
| csv - How to unstack data from one column to multiple column? - Stack Overflow | https://stackoverflow.com/questions/56440446/how-to-unstack-data-from-one-column-to-multiple-column/56441388#56441388 |
| text processing - Convert rows into columns in bash shell script - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/523479/convert-rows-into-columns-in-bash-shell-script/523735#523735 |
| awk - Comparing two .csv files and write into 3rd file - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/524169/comparing-two-csv-files-and-write-into-3rd-file/524212#524212 |
| text processing - CSV to TSV without replacing commas within quotes - Ask Ubuntu | https://askubuntu.com/questions/1153856/csv-to-tsv-without-replacing-commas-within-quotes/1153968#1153968 |
| how to add new column with constant value in a text file - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/527563/how-to-add-new-column-with-constant-value-in-a-text-file/527614#527614 |
| qgis - Finding maximum distribution between multiple columns - Geographic Information Systems Stack Exchange | https://gis.stackexchange.com/questions/328580/finding-maximum-distribution-between-multiple-columns/328617#328617 |
| data cleaning - File format where column names are repeated on each row - Data Science Stack Exchange | https://datascience.stackexchange.com/questions/55879/file-format-where-column-names-are-repeated-on-each-row/55917#55917 |
| csv - Unable to simply use tr to convert a tab-delimited-variable file to a comma-delimited-variable file - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/532043/unable-to-simply-use-tr-to-convert-a-tab-delimited-variable-file-to-a-comma-deli/532119#532119 |
| linux - split file lines based on first field - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/532092/split-file-lines-based-on-first-field/532123#532123 |
| linux - Conditional append of strings on fields in a csv file - Stack Overflow | https://stackoverflow.com/questions/57226479/conditional-append-of-strings-on-fields-in-a-csv-file/57231210#57231210 |
| bash - Joining / merging CSV files that do not share all their headers / column - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/533490/joining-merging-csv-files-that-do-not-share-all-their-headers-column/533601#533601 |
| text processing - Merging multiple CSV files for matching and non matching columns - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/450915/merging-multiple-csv-files-for-matching-and-non-matching-columns/533605#533605 |
| awk - Get difference between two csv files based on column using bash - Stack Overflow | https://stackoverflow.com/questions/57388777/get-difference-between-two-csv-files-based-on-column-using-bash/57389381#57389381 |
| text processing - Extract file with two matched strings at the same time - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/535351/extract-file-with-two-matched-strings-at-the-same-time/535378#535378 |
| Page not found - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/535518/nested-loops-for-text-file/535591#535591 |
| python - Reading Large CSV and splitting it into smaller chunks - Stack Overflow | https://stackoverflow.com/questions/57586507/reading-large-csv-and-splitting-it-into-smaller-chunks/57587030#57587030 |
| Bash, grep lines from csv based on column 3, if length is 6 chars (numbers) - Stack Overflow | https://stackoverflow.com/questions/57672681/bash-grep-lines-from-csv-based-on-column-3-if-length-is-6-chars-numbers/57673347#57673347 |
| python - How to split CSV rows then duplicate that row? - Stack Overflow | https://stackoverflow.com/questions/57874901/how-to-split-csv-rows-then-duplicate-that-row/57877936#57877936 |
| csv - How to flatten this json as a tsv? - Stack Overflow | https://stackoverflow.com/questions/57978050/how-to-flatten-this-json-as-a-tsv/57978379#57978379 |
| text processing - Replace the content of specific cells in a tsv file with the edited content of other cells - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/542539/replace-the-content-of-specific-cells-in-a-tsv-file-with-the-edited-content-of-o/542565#542565 |
| text processing - Given Pattern Replacement With Comma - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/543221/given-pattern-replacement-with-comma/543311#543311 |
| linux - Filtering CSV dataset to find rows and store them in text file - Unix & Linux Stack Exchange | https://unix.stackexchange.com/a/544059/195582 |
| text processing - split the content of a specific column with variable number of strings - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/543726/split-the-content-of-a-specific-column-with-variable-number-of-strings/544063#544063 |
| linux - CSV dataset to text dataset - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/544163/csv-dataset-to-text-dataset/544166#544166 |
| linux - Splitting a column into two columns base on a separator in csv file using Awk? - Stack Overflow | https://stackoverflow.com/questions/58128032/splitting-a-column-into-two-columns-base-on-a-separator-in-csv-file-using-awk/58146864#58146864 |
| bash - Compare 2 cols; when 1st col > 2nd col, remove 2nd col - Ask Ubuntu | https://askubuntu.com/questions/1178129/compare-2-cols-when-1st-col-2nd-col-remove-2nd-col/1178661#1178661 |
| text processing - Count number of different name in a file - Ask Ubuntu | https://askubuntu.com/questions/1178592/count-number-of-different-name-in-a-file/1178739#1178739 |
| text processing - extract columns with certain characters and store them in separate file - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/545343/extract-columns-with-certain-characters-and-store-them-in-separate-file/545356#545356 |
| Create a .csv File from JSON Data Using JQ - Stack Overflow | https://stackoverflow.com/questions/58271333/create-a-csv-file-from-json-data-using-jq/58277093#58277093 |
| shell script - Compare 2 comma separated files and output the difference - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/545788/compare-2-comma-separated-files-and-output-the-difference/545937#545937 |
| linux - How to split a CSV file such that first column stays? - Stack Overflow | https://stackoverflow.com/questions/58327627/how-to-split-a-csv-file-such-that-first-column-stays/58330927#58330927 |
| shell script - Merge CSV Files based on ID even when Unmatched IDS and Indefinite Number of Columns - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/546910/merge-csv-files-based-on-id-even-when-unmatched-ids-and-indefinite-number-of-col/547056#547056 |
| linux - How to edit a cell in a CSV file - Stack Overflow | https://stackoverflow.com/questions/45484459/how-to-edit-a-cell-in-a-csv-file/58412431#58412431 |
| How to parse a complicated csv file in R? - Stack Overflow | https://stackoverflow.com/questions/58431657/how-to-parse-a-complicated-csv-file-with-in-r/58433337#58433337 |
| regex - Edit a CSV column through another column - Stack Overflow | https://stackoverflow.com/questions/58459956/edit-a-csv-column-through-another-column/58461626#58461626 |
| csv - How to import nested json file into google sheets with some missing key:value pairs - Stack Overflow | https://stackoverflow.com/questions/58603534/how-to-import-nested-json-file-into-google-sheets-with-some-missing-keyvalue-pa/58630961#58630961 |
| command line - How to rename the exisitng cell names based on number of their occurnaces in a particular column? - Ask Ubuntu | https://askubuntu.com/questions/1184179/how-to-rename-the-exisitng-cell-names-based-on-number-of-their-occurnaces-in-a-p/1185056#1185056 |
| bash - Remove quotes "..." from all columns with AWK - Stack Overflow | https://stackoverflow.com/questions/58627542/remove-quotes-from-all-columns-with-awk/58638816#58638816 |
| shell script - Sum of even and odd numbers of odd and even lines - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/550676/sum-of-even-and-odd-numbers-of-odd-and-even-lines/550736#550736 |
| python - Transform CSV file into a 'flat file" for pandas - Stack Overflow | https://stackoverflow.com/questions/58786303/transform-csv-file-into-a-flat-file-for-pandas/58790398#58790398 |
| text processing - Combine file data, sort on chromosome, position next to each other - column vise - Unix & Linux Stack Exchange | https://unix.stackexchange.com/questions/551162/combine-file-data-sort-on-chromosome-position-next-to-each-other-column-vise/551417#551417 |
| grel - How to merge rows in OpenRefine - Stack Overflow | https://stackoverflow.com/questions/58677751/how-to-merge-rows-in-openrefine/58814336#58814336 |
| linux - Merge content of two csv file side by side in bash - Stack Overflow | https://stackoverflow.com/questions/58961650/merge-content-of-two-csv-file-side-by-side-in-bash/58964168#58964168 |
| shell - Sorting csv file based on column 2 only. if 2 rows of column 2 are same, should be input file order only. should not sort based on any other column - Stack Overflow | https://stackoverflow.com/questions/59041187/sorting-csv-file-based-on-column-2-only/59041272#59041272 |

**Reply by johnkerl** [reply]
Wow @aborruso this is amazing!!!!!!!!! :D

**Reply by NikosAlexandris** [reply]
I'd go more and add tags. I wanted to do this, but I am hard to find some free-time.
Here with an additional verbs column and a few examples: https://gist.github.com/NikosAlexandris/b0a201b18fd894bc0ce5096cd270ecda

**Reply by NikosAlexandris** [reply]
For example, now that I want to try the suggested join in #608 of course I would search for the `join` string too. Titles and questions may not be that declarative. Given some meta-columns, I could do as well
```
mlr --tsv grep 'join' millerStackExchange.tsv
```
:-)

**Reply by NikosAlexandris** [reply]
> I'd go more and add tags. I wanted to do this, but I am hard to find some free-time.
> Here with an additional verbs column and a few examples: https://gist.github.com/NikosAlexandris/b0a201b18fd894bc0ce5096cd270ecda

I will try to do this once I find some free time. Or, can we have a common place to edit this very file?

**Reply by janxkoci** [reply]
> can we have a common place to edit this very file?

Sounds like a job for Google Sheets. :wink:

**Reply by NikosAlexandris** [reply]
@johnkerl @janxkoci @aborruso I did not find time to work on this, and I am overloaded with work. However, why not continue in https://github.com/aborruso/millerQuestions?

**Comment by NikosAlexandris** [comment]
Perhaps it's also useful to revisit and transfer some of the closed issues labeled with `question` in this table too?
I read all `question`s and I think issues #76, #78, #80, #94 might be worth to save or rework if new stuff provide easier solutions.

**Comment by aborruso** [comment]
@NikosAlexandris I have created a [repo](https://github.com/aborruso/millerQuestions):

- as [source file in YAML](https://github.com/aborruso/millerQuestions/blob/main/data/millerQuestions.yml). I have choosen this format because is both human and machine readable
- two output:
  - CSV https://github.com/aborruso/millerQuestions/blob/main/output/millerQuestions.csv
  - markdown https://github.com/aborruso/millerQuestions/blob/main/output/millerQuestions.md

I can add you as collaborator in the repo or you can make PRs.

For every yaml file update and push, a github action will create updated output
